README.md

```diff
@@ -44,18 +44,20 @@ To use Kintsugi as a Git merge driver, follow these steps:
 
 - Add it as driver to Git config file by running the following:
 ```sh
-git config merge.kintsugi.name "Kintsugi driver" # Or any other name that your prefer
+git config merge.kintsugi.name "Kintsugi driver" # Or any other name you prefer
```

Comment:
reconsider whether the probable use is setting this for a specific repository or making it global.

Reply:
I still prefer to have the local case the default :)